src/parser/pegen.ts

```diff
@@ -2013,6 +2031,29 @@ export function set_expr_context(expr: expr, ctx: expr_context): expr {
 //                         class_def->end_col_offset, p->arena);
 // }
 
+class KeywordOrStarred {
+    element: any;
+    is_keyword: boolean;
+
+    constructor(element: any, is_keyword: boolean) {
+        this.element = element;
+        this.is_keyword = is_keyword;
+    }
+}
+
+// @stu help me out here a void pointer!? aaaahhhhh
+// @stu for this to be a boolean we need to change the grammar aaaahhhhh
+/*
```

Comment:
here's a few more comments that made me very confused

Reply:
Is it that cpython uses 1 and 0 instead of true or false? Or is it something else?

Reply:
The void pointer. But I think that's probably just c's way of doing `any`

Reply:
And whether or not we want to change the grammar for the boolean change